### ISSUE TITLE
Configuring tzdata stops with this interactive dialog

### ISSUE THREAD

**Issue by HediaTnani** [NCBI-Hackathons/Graph_Genomes_CSHL#42]
![image](https://user-images.githubusercontent.com/59562743/109061143-8f14e900-76e6-11eb-90a3-8ae8eaa4b05c.png)
It stays forever at this step. I tried many alternatives `ARG DEBIAN_FRONTEND=noninteractive` and `ENV TZ=Africa/Tunis
RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
`

**Comment by evanbiederstedt** [comment]
Hi @HediaTnani 

Thanks for the issue. 

Are you sure this is the right repository? I don't think we do anything with the TZ database :)

If we do....where is there a prompt that this be configured?

**Comment by evanbiederstedt** [comment]
I have noticed this in the Dockerfile you've created on your repo associated with the project:

https://github.com/HediaTnani/NovoGraph/blob/main/Dockerfile#L5-L8

```
# Set timezone
ENV TZ=Africa/Tunis
RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

This might be useful for you perhaps? https://grigorkh.medium.com/fix-tzdata-hangs-docker-image-build-cdb52cc3360d

Otherwise, you don't need to configure the tz database for Novograph at all.

I'll close this issue, as I think it's something in your Dockerfile you're working with :)

Best, Evan
